tests/android/MASVS-CODE/MASTG-TEST-0x27-2.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: Testing for URL Loading in WebViews
+ID: MASTG-TEST-0x27-2
+Link: https://mas.owasp.org/MASTG/tests/android/MASVS-CODE/MASTG-TEST-0027/
+Platform: android
+type: [dynamic]
+MASVS v1: ['MSTG-PLATFORM-2']
+MASVS v2: ['MASVS-CODE-4']
+---
+
+## Overview
+
+By default, navigation events inside of a WebView will redirect to the default browser application. However, it is possible to stay within the WebView and handle all new page loads. This can be dangerous, as the new page may be malicious and interact with either the JavaScript bridge, or phish the user. The application should monitor navigation events inside the WebView to make sure that only legitimate pages are loaded, while others are redirected to the browser application.
```

Comment:
```suggestion
By default, navigation events inside of a WebView will redirect to the default browser application. However, it is possible to stay within the WebView and handle all new page loads. This can be dangerous, as the new page may be malicious and interact with either the JavaScript bridge (see @MASWE-0068), or phish the user. The application should monitor navigation events inside the WebView to make sure that only legitimate pages are loaded, while others are redirected to the browser application.
```